### PR TITLE
add licenses

### PR DESCRIPTION
**Pull request by QuincyLarson**
Closes https://github.com/freeCodeCamp/guides/issues/538

### CODE REVIEW

File: LICENSE-freeCodeCamp-Guide-Articles.md [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+     reasons, including because others have copyright or other
+     rights in the material. A licensor may make special requests,
+     such as asking that all changes be marked or described.
+     Although not required by our licenses, you are encouraged to
+     respect those requests where reasonable. More_considerations
+     for the public:
+  wiki.creativecommons.org/Considerations_for_licensees
+
+=======================================================================
+
```
Comment (systimotic) [inline LICENSE-freeCodeCamp-Guide-Articles.md]: Everything above this is not a part of the license: "The following considerations (...) do not form part of our licenses."

Should it be included in this file?

File: LICENSE-freeCodeCamp-Guide-Articles.md [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+  c. No term or condition of this Public License will be waived and no
+     failure to comply consented to unless expressly agreed to by the
+     Licensor.
+
+  d. Nothing in this Public License constitutes or may be interpreted
+     as a limitation upon, or waiver of, any privileges and immunities
+     that apply to the Licensor or You, including from the legal
+     processes of any jurisdiction or authority.
+
+
```
Comment (systimotic) [inline LICENSE-freeCodeCamp-Guide-Articles.md]: I believe everything below this is not a part of the license either.

**Comment by QuincyLarson** [comment]
@systimotic I copied that directly from the CC license we're using on github.com/freecodecamp/freecodecamp

If this isn't accurate, could you double check and update this PR, then also raise a PR against the github.com/freecodecamp/freecodecamp license file?

And I agree that we should update the README.md. 

Here's the copy we use for this on freeCodeCamp:

# License

Copyright (c) 2017 freeCodeCamp.

The content of this repository bound by the following LICENSE(S)

The computer software is licensed under the BSD-3-Clause.
The curricular content as in the ./seed/challenges or subdirectories and our wiki are licensed under the CC-BY-SA-4.0


We would just need to change "curricular content" to "reference content" and and update the directory.

**Comment by QuincyLarson** [comment]
@systimotic Awesome! Thanks for confirming this, and for updating the licenses text!

**Pull request merged**